src/app/public/modules/toast/toast.component.ts

```diff
@@ -88,14 +103,46 @@ export class SkyToastComponent implements OnInit {
 
   private isOpen = false;
 
+  private autoCloseTimeoutId: any;
```

Comment:
Code style > alphabetical ordering of private properties.

Reply:
Fixed.